app/grandchallenge/reader_studies/templates/reader_studies/readerstudy_ground_truth.html

```diff
@@ -0,0 +1,74 @@
+{% extends "base.html" %}
+{% load url %}
+{% load static %}
+
+{% block title %}
+    Ground Truth - {{ object }} - {{ block.super }}
+{% endblock %}
+
+{% block breadcrumbs %}
+    <ol class="breadcrumb">
+        <li class="breadcrumb-item"><a href="{% url 'reader-studies:list' %}">Reader
+            Studies</a></li>
+        <li class="breadcrumb-item"><a href="{{ object.get_absolute_url }}">{{ object }}</a></li>
+        <li class="breadcrumb-item active" aria-current="page">Ground Truth</li>
+    </ol>
+{% endblock %}
+
+{% block content %}
+
+<h2>Ground Truth</h2>
```

Comment:
The indentation doesn't seem correct, should be 4 spaces over as it's inside a block.